src/index.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import './index.css';
+import App from './App';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <App />
+  </React.StrictMode>,
```

Comment:
- [ ] I had some issues running your app. The first error I got was this (see image). I fixed it by wrapping the app with Provider from `react-redux` and importing the store from the config file.

  ```
  <React.StrictMode>
    <Provider store={store}>
      <App />
    </Provider>
  </React.StrictMode>,
  ```

  ![kfof](https://user-images.githubusercontent.com/54848363/196523629-f30ae334-49cb-4e0d-aee9-e73ea834124b.jpg)


- [ ] After that I got this error next. Kindly look into that and fix it.